uptane-whitepaper.md

```diff
@@ -1 +1,106 @@
+
+## **What are the threats and why isn’t signing enough?**
+Increasingly, software updates for automotive ECUs are done using ["software-over-the-air”](https://ihsmarkit.com/research-analysis/remote-software-update-future-growth-business.html)or SOTA strategies, in which revised versions of programs are sent directly to computing units through broadcasts over the Internet. The strategy is growing in popularity because it is a much quicker and more cost-effective delivery mechanism than traditional methods, such as distributing flash drives or requiring customers to bring in their vehicles for servicing. Established OEMs are recognizing the potential of this strategy. In May 2019, GM In May 2019, GM [announced](https://www.theverge.com/2019/5/21/18633000/gm-ota-software-updates-digital-platform-reuss)  that the 2020 Cadillac CT5 sedan will feature over-the-air software update, and that such systems will be standard on all GM vehicles in the next four years. Likewise, Ford’s website  announced that it would begin equipping most redesigned vehicles in the U.S. with advanced over-the-air update (OTA) beginning in 2020, starting with the all-electric [Mustang Mach-E](https://media.ford.com/content/fordmedia/fna/us/en/news/2020/05/12/new-ford-over-the-air-updates-mustang-mach-e.html).
+
+Unfortunately,connecting ECUs to the Internet make them vulnerable to a wide range of attacks, some of which, like SolarWinds, introduce malware masquerading as legitimate updates. The potential consequences of such attacks could be costly indeed, not only in terms of recalls or lost sales, but also potentially, in loss of life.
+
+The types of attacks an automotive software security system needs to to defend against can be organized into four categories, presented here in order of increasing severity.
+
+**Read updates:** The goal here is intellectual property theft, so these attackers aim to read the contents of software updates. This is generally achieved with an Eavesdrop attack, where attackers can read unencrypted updates sent from the repository to the vehicles.
+
+**Deny updates:** In this group of attacks, the goal is to deny access to updates so vehicles cannot fix software problems. These strategies include:
+- *Drop-request attack:* blocks network traffic outside or inside the vehicle to prevent an ECU from receiving any updates.
+- *Slow retrieval attack:* slows delivery time of updates to ECUs so a known security vulnerability can be exploited before a corrective patch is received.
+- *Freeze attack:* continues to send the last known update to an ECU, even if a newer update exists.
+- *Partial bundle installation attack:* Allows only part of an update to install by dropping traffic to selected ECUs.
+
+**Deny functionality:** This grouping ups the threat ante a bit further by causing vehicles to fail to function in one of the following ways:
+- *Rollback attack:* tricks an ECU into installing outdated software with known vulnerabilities.
```

Comment:
I'd say a rollback attack fits a bit better under control. It can be used to install any previously valid image.

Reply:
Ok. I will move that.

Reply:
Done